src/components/molecules/Dropdown.js

```diff
@@ -2,8 +2,10 @@ import React from "react";
 import styled from "styled-components";
 
 const Drop = styled("select")`
-    width: 100%;
-    height: 25px !important;
+	&&&& {
```

Comment:
Why four here but two everywhere else?

Reply:
Thats how many levels it required for the styling to take effect. The amount is situational and not always constant.